content/applications/hr/payroll/work_entries.rst

```diff
@@ -2,75 +2,86 @@
 Work entries
 ============
 
-The *Work Entries* dashboard, which can be found by going to :menuselection:`Payroll --> Work
-Entries --> Work Entries`, provides a visual overview of the individual time sheets of every
-employee, with each day split into a morning shift and an afternoon shift.
+Work entries are created automatically in the *Payroll* app, based on the employee's :ref:`salary
+structure type <payroll/structure-types>`, and from the *Planning*, *Attendances*, and *Time Off*
+applications.
+
+The *Work Entries* dashboard of the *Payroll* application provides a visual overview of the
+individual work entries for every employee.
+
+To open the dashboard, navigate to :menuselection:`Payroll app --> Work Entries --> Work Entries`.
+
+On the :guilabel:`Work Entry` dashboard, work entries appear in alphabetical order, based on the
+first name of the employees. The entire month is displayed, with the current day highlighted in pale
+yellow.
+
+If any entries have :ref:`conflicts <payroll/conflicts>` that need to be resolved, the dashboard
+defaults to filter only the `Conflicting` entries.
+
+To remove the filter from the :guilabel:`Search` bar to view all work entries, click the
+:guilabel:`✖️ (remove)` icon on the `Conflicting` filter in the :guilabel:`Search` bar, and all
+work entries appear in the list.
```

Comment:
Please match the text on the UI exactly when using :guilabels: (:guilabel:\`Search\` ➡️ :guilabel:\`Search...\`)
```suggestion
To remove the filter from the search bar to view all work entries, click the :guilabel:`✖️ (remove)`
icon on the :guilabel:`Conflicting` filter in the :guilabel:`Search...` bar, and all work entries
appear in the list.
```
or
```suggestion
To remove the filter from the :guilabel:`Search...` bar to view all work entries, click the
:guilabel:`✖️ (remove)` icon on the :guilabel:`Conflicting` filter in the :guilabel:`Search...` bar,
and all work entries appear in the list.
```

Reply:
Ahhh, ok. When KSC did my last review, he changed it to the GUI 'Search...' like you have in the second option, but then changed it in a 2nd review to remove the "...".  I do agree with you , that is how it appears, so that's how it should look. I'll see if we can add this to our linter/style guide so we can be consistent.